runhouse/cli_utils.py

```diff
@@ -471,12 +471,14 @@ def print_bring_cluster_up_msg(
     )
 
 
-def get_cluster_or_local(cluster_name: str = None):
+def get_cluster_or_local(cluster_name: str = None, cmd_is_up: bool = False):
     from runhouse.main import console
 
     if cluster_name:
         current_cluster = rh.cluster(name=cluster_name, dryrun=True)
         if not current_cluster.is_up():
+            if cmd_is_up:
```

Comment:
what was the issue with leaving as we had before if running `up`?

Reply:
we were just printing an error msg that the cluster is not up, and exiting, without returning the cluster object. For the `up` command, we don't need to print the error msg, just return the cluster object so we could bring it up.